include/internal/yamerl_constr.hrl

```diff
@@ -35,6 +35,7 @@
 
 -type schema() :: failsafe | json | core | yaml11.
 -type yamerl_constr_option() :: {detailed_constr, boolean()}
+                              | {allow_duplicate_keys, boolean()}
```

Comment:
I'm not sure about the naming. Duplicate keys are allowed, regardless of that option (i.e. they are not rejected by the parser or the constructor).

The intent here is to *keep* them as multiple entries in the constructed term if possible. So I would call it `keep_duplicate_keys`. What do you think?

Reply:
I agree!